share/lib/python/neuron/rxd/rxd.py

```diff
@@ -547,8 +533,8 @@ def _cxx_compile(formula):
         sysconfig.get_path("include"),
         os.path.join(h.neuronhome(), "..", "..", "include", "nrn"),
```

Comment:
I think this include path can be removed, it was there to find the `rxdmath.h` file

Reply:
Removed, works. Good catch.